src/unleash.ts

```diff
@@ -84,9 +93,21 @@ export class Unleash extends EventEmitter {
     bootstrapOverride,
     storageProvider,
     disableAutoStart = false,
+    skipInstanceCountWarning = false,
   }: UnleashConfig) {
     super();
 
+    Unleash.instanceCount++;
+
+    this.on(UnleashEvents.Error, (err) => {
+      console.error(err?.message);
```

Comment:
we need to check if there are other event handlers registered and only console.error if no others have registered.